assets/apps/calendar/init.js

```diff
@@ -282,58 +560,124 @@ function ScheduleInfo() {
         }
     };
 }
-
-function removeScheduleFromCalendar(schedule) {
-    cal.deleteSchedule(schedule.id, schedule.calendarId);
-    ScheduleList.splice(ScheduleList.findIndex(v => v.id === schedule.id), 1);
-    removeFromCache(schedule);
-    deleteSchedule(schedule);
+function addEXDateToSchedule(scheduleId, startDate) {
+    var comp = LoadedEvents[scheduleId];
+    let vevents = comp.getAllSubcomponents('vevent');
+    let vvent = vevents[0];
+    let exDate = toICalTimeTZ(comp, startDate, false);
+    vvent.addPropertyWithValue('exdate', exDate);
+}
+function untilUTCTimeString(momentJS) {
+    let currentTZ = momentJS.clone().tz(getCurrentTimeZoneId());
+    let dateParts = currentTZ.toISOString().split('-').join('').split(':').join('');
+    let formattedDate = dateParts.substring(0, dateParts.indexOf('.')) + "Z";//FIXME TODO not correct format according to spec
+    return formattedDate;
+}
+//FIXME not according to spec
```

Comment:
maybe document what the difference is from the spec?

Reply:
done